core/templates/pages/exploration-editor-page/services/autosave-info-modals.service.ts

```diff
@@ -72,24 +61,12 @@ angular.module('oppia').factory('AutosaveInfoModalsService', [
             'save-version-mismatch-modal.template.html'),
           // Prevent modal from closing when the user clicks outside it.
           backdrop: 'static',
-          controller: ['$scope', function($scope) {
-            // When the user clicks on discard changes button, signal backend
-            // to discard the draft and reload the page thereafter.
-            $scope.discardChanges = function() {
-              ExplorationDataService.discardDraft(function() {
-                _refreshPage(20);
-              });
-            };
-
-            $scope.hasLostChanges = (lostChanges && lostChanges.length > 0);
-            if ($scope.hasLostChanges) {
-              // TODO(sll): This should also include changes to exploration
-              // properties (such as the exploration title, category, etc.).
-              $scope.lostChanges = lostChanges.map(
-                LostChangeObjectFactory.createNew);
-              $log.error('Lost changes: ' + JSON.stringify(lostChanges));
+          resolve: {
+            lostChanges: function() {
+              return lostChanges;
```

Comment:
```suggestion
           lostChanges: () => lostChanges
```

In order to match other files.

Reply:
Ditto below!

Reply:
Done.